Constitution.md

```diff
@@ -171,17 +171,35 @@ The Committee may appoint as Honorary President and Vice-President, any persons
 5. Nominations must be made in accordance with instruction officially communicated by the Committee.
-6. The voting system shall be a single vote for each position, with a simple majority.
-7. The Chairperson of the General Meeting to have the casting vote in the event of a tie, having not voted previously.
+6. Voting must be by secret, secure ballot either through paper or electronic means.
+7. The default system of voting shall be First Past the Post ("FPTP"), for ease of counting only. Where FPTP is used as the system of voting, it shall be defined as follows.
+   1. Eligible voters shall cast (up to) the same number of votes as there are positions to be filled.
+   2. Each vote shall be a mark next to the selected candidate(s).
+   3. In a multi-position election, each voter may not vote more than once for any candidate.
+   4. The number of votes received for each candidate is counted, and the highest-polling candidate(s) corresponding to the number of positions to be filled are elected.
+8. However, it is recognized that the Single Transferable Vote ("STV") system of voting is typically a fairer way of electing candidates, though requires a more intensive counting process. If one or more of the following apply prior to the commencement of each election, then STV must be used as the system of voting for that election instead of FPTP.
+
+   1. The Chairperson decides in their sole authority to require the use of the STV system of voting for that election, and/or
+   2. Any eligible voter requests the use of the STV system of voting for that election, provided that voter can demonstrate support from a number of eligible voters (including themselves) of no less than 10% of the quorum threshold (as defined by 8(d)). For the avoidance of doubt, this refers to 10% of threshold required for quoracy, not 10% of the number of eligible voters present.
+
+   These criteria shall apply on a per-role basis, so different systems of voting may be used for each election within one General Meeting. There shall be no limit as to the number of elections conducted using either system of voting, provided the criteria for doing so are duly observed.
+
+9. The Chairperson must give eligible voters a reasonable opportunity to request the use of STV prior to the commencement of each election. Provided this reasonable opportunity is duly given, it shall not be permitted to change retrospectively the system of voting after the commencement of that election.
+10. In the event that the STV system of voting is used, it shall be used as defined by the Electoral Reform Society of Great Britain and Northern Ireland ('How to conduct an election by the Single Transferable Vote', 3rd Edition, June 1997).
+11. The system of voting used for each election shall be recorded in the minutes of the General Meeting. Where STV is used as the system of voting, it shall also be recorded which criterion (or criteria) applied prior to the commencement of that election.
```

Comment:
11. I would be tempted to reword `criterion (or criteria)` to better emphasize that the criterion is related to the selection of STV as a voting system.

Reply:
added ``which caused STV to be used``